packaging/installer/kickstart.sh

```diff
@@ -1468,8 +1470,17 @@ try_package_install() {
   fi
 
   repoconfig_name="netdata-repo${release}"
-  repoconfig_file="${repoconfig_name}${pkg_vsep}${REPOCONFIG_VERSION}${pkg_suffix}.${pkg_type}"
-  repoconfig_url="${REPOCONFIG_URL_PREFIX}/${repo_prefix}/${repoconfig_file}/download.${pkg_type}"
+
+  case "${pkg_type}" in
+    deb)
+      repoconfig_file="${repoconfig_name}${pkg_vsep}${REPOCONFIG_DEB_VERSION}${pkg_suffix}.${pkg_type}"
+      repoconfig_url="${REPOCONFIG_DEB_URL_PREFIX}/${repo_prefix}/${repoconfig_file}/download.${pkg_type}"
+      ;;
+    rpm)
+      repoconfig_file="${repoconfig_name}${pkg_vsep}${REPOCONFIG_RPM_VERSION}${pkg_suffix}.${pkg_type}"
+      repoconfig_url="${REPOCONFIG_RPM_URL_PREFIX}/${repo_prefix}/${repoconfig_file}/${repoconfig_file}"
```

Comment:
I think `repoconfig_url` is wrong, please check the log when I was testing on Fedora 37:
```
 --- Checking for availability of repository configuration package. ---
+ check_for_remote_file https://repo.netdata.cloud/repos/repoconfig//fedora/37/netdata-repo-edge-2-1.no
arch.rpm/netdata-repo-edge-2-1.noarch.rpm
+ url=https://repo.netdata.cloud/repos/repoconfig//fedora/37/netdata-repo-edge-2-1.noarch.rpm/netdata-r
epo-edge-2-1.noarch.rpm
```

Reply:
The _problem_ is `REPOCONFIG_RPM_URL_PREFIX`,  it has trailing `/` (`REPOCONFIG_DEB_URL_PREFIX` has no).